packages/k2-alpine/src/theme/tokens/Colors.stories.tsx

```diff
@@ -40,79 +40,84 @@ const Color = ({
-        )}
-        numColumns={2}
-      />
-      <Text variant="heading6">Dark Mode</Text>
-      <FlatList
-        style={{
-          width: '100%',
-          paddingVertical: 10,
-          marginVertical: 20
-        }}
-        contentContainerStyle={{ gap: 10, marginHorizontal: 10 }}
-        columnWrapperStyle={{ gap: 10 }}
-        data={Object.entries(darkModeColors)}
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        renderItem={({ item }: { item: any }) => (
-          <Color key={item[0]} name={item[0]} value={item[1]} />
-        )}
-        numColumns={2}
-      />
-    </ScrollView>
+        gap: 12,
+        padding: 12,
+        backgroundColor: lightModeColors.$surfacePrimary
+      }}
+      data={Object.entries(lightModeColors)}
+      ListHeaderComponent={
+        <View style={{ alignItems: 'center' }}>
+          <Text
+            variant="heading3"
+            style={{ color: lightModeColors.$textPrimary }}>
+            Light Mode
+          </Text>
+          <Link
+            title="Figma Source"
+            url={FIGMA_LINK}
+            style={{
+              marginVertical: 20,
+              color: lightModeColors.$textPrimary
+            }}
+          />
+        </View>
+      }
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      renderItem={({ item }: { item: any }) => (
+        <Color key={item[0]} name={item[0]} value={item[1]} />
+      )}
+    />
+  )
+}
+
+export const DarkMode = (): JSX.Element => {
+  return (
+    <FlatList
+      style={{ flex: 1, width: '100%' }}
+      contentContainerStyle={{ gap: 12, padding: 12 }}
+      data={Object.entries(darkModeColors)}
+      ListHeaderComponent={
+        <View style={{ alignItems: 'center' }}>
+          <Text variant="heading3">Dark Mode</Text>
+          <Link
+            title="Figma Source"
+            url={FIGMA_LINK}
+            style={{ marginVertical: 20 }}
+          />
+        </View>
+      }
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      renderItem={({ item }: { item: any }) => (
+        <Color key={item[0]} name={item[0]} value={item[1]} />
+      )}
+    />
   )
 }
 
 export const Colors = (): JSX.Element => {
   return (
-    <ScrollView
-      style={{ width: '100%' }}
-      contentContainerStyle={{
-        top: '5%',
-        alignItems: 'center',
-        paddingBottom: '15%'
-      }}>
-      <Link
-        title="Figma Source"
-        url={FIGMA_LINK}
-        style={{ marginVertical: 20 }}
-      />
-      <FlatList
-        style={{ width: '100%', marginTop: 20 }}
-        contentContainerStyle={{ gap: 10, marginHorizontal: 10 }}
-        data={Object.entries(colors)}
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        renderItem={({ item }: { item: any }) => (
-          <Color key={item[0]} name={item[0]} value={item[1]} />
-        )}
-      />
-    </ScrollView>
```

Comment:
`FlatList` was nested in `ScrollView`, which is an anti-pattern and causes a lot of warnings.